plf-tomcat-resources/src/main/resources/stop_eXo.sh

```diff
@@ -66,6 +66,8 @@ while [ "$1" != "" ]; do
     shift
 done
 
-export CATALINA_PID=${CATALINA_PID:-"$PRGDIR/temp/catalina.pid"}
+if [ -x "$PRGDIR/temp/catalina.pid" ]; then
+  export CATALINA_PID=${CATALINA_PID:-"$PRGDIR/temp/catalina.pid"}
+fi
```

Comment:
What is the purpose of this change ?
If the file $PRGDIR/temp/catalina.pid does not exist, the CATALINA_PID variable will keep its current value which can be a custom location where the catalina.pid does not exist also.

Reply:
If the catalina PID doesn't exist, the stop process will exit without calling stop using Tomcat stop port (8005 by default).
The error will be:
```
Using CATALINA_PID:    ./temp/catalina.pid
$CATALINA_PID was set but the specified file does not exist. Is Tomcat running? Stop aborted.

```

Reply:
Yes, but this change also prevents the stop to abort even if we set a different location in  CATALINA_PID ?

Reply:
changed to use variable too